src/components/Editor/ColumnBreakpoint.js

```diff
@@ -26,10 +28,10 @@ type Props = {
 const breakpointImg = document.createElement("div");
 function makeBookmark(isActive, { onClick }) {
   const bp = breakpointImg.cloneNode(true);
-  bp.className = classnames("call-site", {
-    active: isActive
-  });
+  const className = isActive ? "active" : "disabled";
+  bp.className = classnames("call-site", className);
   bp.onclick = onClick;
+  ReactDOM.render(<Svg name={"column-marker"} />, bp);
```

Comment:
I think for this to be fast we'll need to keep the SVG so we can set stroke and fill, but do what we do in the Gutter w/ breakpoints and have one SVG and clone it every time

https://github.com/devtools-html/debugger.html/blob/master/src/components/Editor/Breakpoint.js#L17-L18